Capfile

```diff
@@ -0,0 +1,31 @@
+###########################################
+# Capistrano Defaults
+########################################### 
+
+# Load DSL and set up stages
+require "capistrano/setup"
+
+# Include default deployment tasks
+require "capistrano/deploy"
+
+require "capistrano/scm/git"
+install_plugin Capistrano::SCM::Git
+
+
+###########################################
+# Chapman Blogs Custom Config
+# https://github.com/chapmanu/blogs
+###########################################   
+
+# Custom Helpers
+require './lib/helpers.rb'
+
+# Include custom strategy for deploying git submodules
+require './lib/capistrano/submodule_strategy'
+
+# Enter Sudo password for restarting nginx/php
+require 'sshkit/sudo'
```

Comment:
Not in the blogs. Added it to counter the issue where it just pauses and doesn't move through to next code block. I'll see what I can do.